app/templates/components/gh-posts-list-item.hbs

```diff
@@ -17,6 +17,8 @@
         {{/if}}
 
         by <span class="gh-content-entry-author">{{authorName}}</span> &mdash;
-        Last edited {{gh-format-timeago post.updatedAtUTC}}
+        Last edited {{gh-format-timeago post.updatedAtUTC}} &mdash;
+        <span class="gh-content-status-published">Created</span> by <span class="gh-content-status-published">{{postDate}}</span>
```

Comment:
I _think_ `by` should be used for the author name, while `on` should be used for times...